crates/matrix-sdk/src/client/mod.rs

```diff
@@ -169,6 +170,7 @@ pub(crate) struct ClientInner {
     pub(crate) event_handlers: EventHandlerStore,
     /// Notification handlers. See `register_notification_handler`.
     notification_handlers: RwLock<Vec<NotificationHandlerFn>>,
+    pub(crate) sync_gap_broadcast_txs: StdMutex<BTreeMap<OwnedRoomId, Observable<()>>>,
```

Comment:
We should probably settle one one single way of doing these broadcasts everywhere instead of mixing tokio and eyeball